docs/02-about-us/guilds.md

```diff
@@ -0,0 +1,44 @@
+# Agile Guilds
+
+As [described in our continous learning page](../04-how-we-work/continuous-learning.md), CivicActions leverages the 
+concept of an Agile Guild to harness organizational feedback from team members and project retrospectives.
+
+## Governance of a Guild
```

Comment:
We could look to add some "Guild Inputs" section, like retrospectives, all-hands calls, retreat sessions. I think some people may not instinctively post to a guild board